controllers/limitador_controller_test.go

```diff
@@ -40,6 +40,23 @@ var _ = Describe("Limitador controller", func() {
 	version := LimitadorVersion
 	httpPort := &limitadorv1alpha1.TransportProtocol{Port: &httpPortNumber}
 	grpcPort := &limitadorv1alpha1.TransportProtocol{Port: &grpcPortNumber}
+	affinity := &v1.Affinity{
+		PodAntiAffinity: &v1.PodAntiAffinity{
+			PreferredDuringSchedulingIgnoredDuringExecution: []v1.WeightedPodAffinityTerm{
+				{
+					Weight: 100,
+					PodAffinityTerm: v1.PodAffinityTerm{
+						LabelSelector: &metav1.LabelSelector{
+							MatchLabels: map[string]string{
+								"app.kubernetes.io/name": "limitador",
+							},
+						},
```

Comment:
Does the conversation around labels and not allowing the user to set the selector field from https://github.com/Kuadrant/limitador-operator/pull/91#discussion_r1293883926 also apply here ? 🤔

Reply:
Not really, in https://github.com/Kuadrant/limitador-operator/pull/91#discussion_r1293883926 the labels are used to select the PDB CR and that is not the case here. That was a test case so I have updated the wording in the label. When using Pod Affinity you are targeting a node to scheduled the pod on. In the test case I use `PodAntiAffinity` rules so it would be trying to add the pod to a node where there is not other pods with the label `pod: label`.

The affinity rules can target Node or Pod using Affinity or AntiAffinity rules. The limitador operator does not make any assumptions on what the data should be in this case. That choice is left up to the SRE teams deploying the infrastructure.